File: isPrime.swift
import Foundation

func isPrime(_ n: Int) -> Bool{
    if n<=1{
        return false
    }
    for i in 2...Int(Double(n).squareRoot()){
        if n%i == 0{
            return false
        }
    }
    return true
}

if let input = readLine(), let number = Int(input){
    if isPrime(number){
        print("Yes")
    }else{
        print("No")
    }
}

File: isPrime.c
#include <stdio.h>
#include <stdbool.h>
#include <math.h>

bool isPrime(int n){
    if (n<=1) return false;
    for (int i =2 ; i*i<n; i++){
    if(n%i == 0) return false;
    }
    return true;
}

int main(){
    int N;
    scanf("%d", &N);

    if(isPrime(N))
    printf("Yes\n");
    else
    printf("No\n");

    return 0;
}